From 7a03af9d14ab38d82254adca9f7c7dc4921f9831 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Sat, 22 Jun 2024 15:58:06 -0700
Subject: [PATCH] Docs/process/index: Remove riscv/patch-acceptance from 'Other
 material' section

'patch-acceptance' on 'Other material' section of 'process/index', which
is for unsorted documents, is actually well organized under
'arch/riscv/' directory, and linked on the index document of the
directory.  Remove it from the 'Other material' section.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/process/index.rst | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Documentation/process/index.rst b/Documentation/process/index.rst
index f66f9cbd0300..08b4eb1e9118 100644
--- a/Documentation/process/index.rst
+++ b/Documentation/process/index.rst
@@ -115,7 +115,6 @@ lack of a better place.
 
    magic-number
    clang-format
-   ../arch/riscv/patch-acceptance
 
 .. only::  subproject and html
 
-- 
2.39.2

